pkg/pillar/agentlog/agentlog.go

```diff
@@ -79,13 +79,17 @@ func handleSignals(sigs chan os.Signal) {
 				log.Warnf("SIGUSR1 triggered with %d stacks", len(stackArray))
 				for _, stack := range stackArray {
 					log.Warnf("%v", stack)
+					// This goes to /persist/agentdebug/<agentname>/stdout file
+					fmt.Printf("%v", stack)
```

Comment:
I would actually have a separate agentname/sigusr1 file for this to make it easier to find.

Reply:
ok

Reply:
done